Pipfile.lock

```diff
@@ -848,6 +848,14 @@
             "markers": "python_version >= '2.7' and python_version not in '3.0, 3.1, 3.2, 3.3'",
             "version": "==0.13.1"
         },
+        "pre-commit": {
```

Comment:
I’m not sure this should be in the lock file… Did this get added automatically?

Reply:
no, it was added manually. thought it was kind of `requirements.txt` equivalent, but probably you are right, it is not required

Reply:
The `Pipfile` is where you put in dependencies, like requirements.txt, `Pipfile.lock` is autogenerated by running `pipenv lock`. But I wouldn't put pre-commit in there since it's not required for the build.